src/pymodaq_plugins_keithley/daq_viewer_plugins/plugins_0D/daq_0Dviewer_Keithley2100.py

```diff
@@ -152,22 +112,20 @@ def grab_data(self, Naverage=1, **kwargs):
         =============== ======== ===============================================
 
         """
+        logger.info("grab_data called")
         data = self.controller.read()
         dte = DataToExport(
             name="K2100",
             data=[
-                DataFromPlugins(
-                    name="K2100", data=data, dim="Data0D", labels=["dat0", "data1"]
-                )
+                DataFromPlugins(name="K2100", data=data, dim="Data0D", labels=["dat0", "data1"])
             ],
         )
 
         self.dte_signal.emit(dte)
 
     def stop(self):
         """Stop the current grab hardware wise if necessary"""
-        self.emit_status(ThreadCommand("Update_Status", ["Acquisition stopped"]))
-        return ""
+        raise NotImplemented   
```

Comment:
Do not raise the error otherwise you'll have issue... just return the empty string